com.unity.shadergraph/Editor/Drawing/Blackboard/BlackboardProvider.cs

```diff
@@ -416,12 +439,12 @@ void DirtyNodes()
             foreach (var node in m_Graph.GetNodes<PropertyNode>())
             {
                 node.OnEnable();
-                node.Dirty(ModificationScope.Node);
+                node.Dirty(ModificationScope.Topological);
```

Comment:
I don't like this, as it requires rebuilding all the previews when we just want to redraw the name. Is there somewhere in the node view we can dirty it instead? Or do we need to regen everything with the new names? Maybe you're right, just checking.

Reply:
bide.Dirty with a ModificationScope.Node _used to_ redraw the node with the new name but doesn't seem to anymore. Temporarily bumping this to topological will resolve that issue while I dig into why, or we can leave it and I can keep digging.

Reply:
Changed the scope back but made the node itself redraw the name on Node scope changes.